lib/rules/no-array-prototype-extensions.js

```diff
@@ -159,7 +177,7 @@ module.exports = {
       recommended: false,
       url: 'https://github.com/ember-cli/eslint-plugin-ember/tree/master/docs/rules/no-array-prototype-extensions.md',
     },
-    fixable: null,
+    fixable: 'code',
```

Comment:
As @bmish has called out, this is a known risk as the rule is not 100% perfect. In case there are multiple ways of fixing an issue, we can still apply some default auto fix and a user can change it as needed. wdyt @smilland ?

Reply:
sounds good!